intellij/src/saros/intellij/editor/LocalEditorManipulator.java

```diff
@@ -243,13 +246,8 @@ public void adjustViewport(@NotNull Editor editor, LineRange range, TextSelectio
       remoteEndLine = range.getStartLine() + range.getNumberOfLines();
 
     } else {
-      int startOffset = selection.getOffset();
-      int endOffset = selection.getOffset() + selection.getLength();
-
-      LineRange selectionRange = EditorAPI.getLineRange(editor, startOffset, endOffset);
-
-      remoteStartLine = selectionRange.getStartLine();
-      remoteEndLine = selectionRange.getStartLine() + selectionRange.getNumberOfLines();
+      remoteStartLine = selection.getStartPosition().getLineNumber();
```

Comment:
`selection` may be null here

Reply:
But if the range is not `null`, the `if` case in line 244 is used, is it not?

Reply:
Ok eclipse shows just a potential npe and github collapsed the diff in a excellent way to show it as possible. nice :D